OsmAnd/res/values/strings.xml

```diff
@@ -11,21 +11,21 @@
+    <string name="monitoring_min_speed_descr_recommendation">Try using motion detection via the "Minimum logging displacement" filter (B) first. It may produce better results, and you will lose less data. If your tracks remain noisy at low speeds, try non-zero values here. Please note, some measurements may not report any speed value at all (some network-based methods), in which case you would not record anything.</string>
+    <string name="monitoring_min_speed_descr_remark">"Speed > 0" check: Most GPS chipsets only report a speed if determining you are in motion. Hence using the "> 0" setting in this filter in a sense uses the motion detection of the GPS chipset. Even if not filtered in OsmAnd at recording time, it still uses this feature GPX analysis to determine the \"Distance corrected\", i.e. the value displayed in that field is the "distance recorded while in motion".</string>
+    <string name="monitoring_min_accuracy_descr">This will record only points measured with a minimum "accuracy" indication (in meters/feet, as reported by Android for your chipset). \"Accuracy\" refers to the scatter of repeated measurements, and is not directly related to precision, which defines how close your measurements are to your \"true\" position.</string>
+    <string name="monitoring_min_accuracy_descr_side_effect">As a result of filtering by accuracy, points may not be recorded below bridges, under trees, between high buildings, or with certain weather conditions.</string>
+    <string name="monitoring_min_accuracy_descr_recommendation">It is hard to predict what will be recorded, it may be best to turn this filter off.</string>
+    <string name="monitoring_min_accuracy_descr_remark">If GPS is only on right before a recording, first point accuracy may decrease, so the OsmAnd strategy may be to wait a second or so before recording a point (or take the median of 3 consecutive points, etc.) in the future.</string>
+    <string name="monitoring_min_distance_descr">This filter avoids duplicate point recording for insufficient actual motion, for a nicer spatial appearance of tracks not post-processed later.</string>
+    <string name="monitoring_min_distance_descr_side_effect">Periods at rest are not recorded, or one point each. Small (real world) movement (e.g. sideways, to mark a possile turnoff on your trip) may be filtered out. Your file contains less info for post-processing, and has worse stats by filtering out superfluous points when recording, while potentially keeping artifacts caused by bad reception or GPS chipset effects.</string>
+    <string name="monitoring_min_distance_descr_recommendation">A setting of 5 meters may work well if you don\'t require any finer details, and don\'t want accuracy in determined stops.</string>
```

Comment:
Honestly, I prefer the original description. It is less technical, but precisely describes what will happen and the side effects.

Also: There are logical weaknesses in your suggestion, like "This is a low-speed cut-off filter smoothing recorded tracks discarded track points..." is not accurate. Yes, the track smoothing is the result, but what will actually happen is simply that no points are being recorded below the minimum speed, and that's what the original description state well enough. Why change it`?

(I think there is also a word missing in line 18.)

Reply:
Fixed 18. I went for shorter, less technical and complex. It really is a pain trying to understand and translate the originals with those compounding problems. What happens is the worst strings end up untranslated (or erroneous like my attempt here), which is a disaster for users that rely on translations.

"Discarded" is what signifies no points recording below threshold. The rewritten attempt is very much clearer following your suggestion.

For someone not well versed in GPS, telling what happens, and then what it does to do that is a lot more clear than the technical info up front.

At first I thought code annotation had been imported into the strings by accident.